javatests/arcs/showcase/imports/BUILD

```diff
@@ -0,0 +1,31 @@
+load(
+    "//third_party/java/arcs/build_defs:build_defs.bzl",
+    "arcs_kt_android_test_suite",
+)
+
+licenses(["notice"])
+
+package(default_visibility = ["//java/arcs:allowed-packages"])
+
+arcs_kt_android_test_suite(
+    name = "imports-tests",
```

Comment:
Just call this `"imports"`, our convention is to name it the same as the package name (i.e. folder name)